hail/python/test/hail/ggplot/test_ggplot.py

```diff
@@ -107,3 +107,12 @@ def test_faceting():
     pfig = (ggplot(ht) + geom_point(aes(x=ht.idx, y=ht.idx)) + facet_wrap(vars(ht.x))).to_plotly()
 
     assert(len(pfig.layout.annotations) == 2)
+
+def test_matrix_table():
+    mt = hl.utils.range_matrix_table(3, 3)
+    mt = mt.annotate_entries(entry_idx = mt.row_idx + mt.col_idx)
+    data = (ggplot(mt, aes(x=mt.row_idx, y=mt.entry_idx)) + geom_point()).to_plotly().data[0]
+    expected = [(0,0),(0,1),(0,2),(1,1),(1,2),(1,3),(2,2),(2,3),(2,4)]
+    for idx, (x, y) in enumerate(zip(data.x, data.y)):
+        assert(expected[idx][0] == x)
+        assert(expected[idx][1] == y)
```

Comment:
could we add a test that we get the right output for non-entry-indexed fields? `mt.row_idx` and `mt.row_idx * 2` for instance (I want to make sure we get 3 points, not 9)?

Also, a sneaky way this testing pattern can be violated is if the lengths of `expected` isn't the same as `data.x` or `data.y`. Let's add an assertion that the lengths of all three agree.